src/pages/home/Home.tsx

```diff
@@ -61,9 +44,18 @@ const Home = () => {
         <section className="category-section">
           <h1 className="text-center text-xl">Categories</h1>
           <div className="category-grid" id="categories">
-            {categoryData.map((category, index) => {
-              return <CategoryCard key={index} category={category} />; //TODO:Give a valid key
-            })}
+            {isLoading && <Loading />}
+            {categoryData?.map(
+              (category: {
+                id: string;
+                categoryDescription: string;
+                categoryImage: string;
+                categoryName: string;
+                categoryStatus: string;
+              }) => {
```

Comment:
As suggested by Charan you could extract these types as well

Reply:
Sure I will refactor in the next pr. Thanks for the review!!